### ISSUE TITLE
LCD connector

### ISSUE THREAD

**Issue by charlespax** [PaxInstruments/t400-electronics#162]
The current LCD connector is not very good for hand soldering. It would be nice to find a better one. We may alternatively change the footprint, giving a larger pad such that a soldering iron can transfer heat to the pad.

It would also be nice to have a flip top version rather than one that is pressed in parallel to the board.

**Comment by charlespax** [comment]
We also discussed a flip top connector, but I think it's not available in this pin pitch.

If we change the footprint such that the pads are more exposed, we should be able to hand solder more easily. @samchoy88 did you change the footprint?

**Comment by samchoy88** [comment]
I made change for Electronics version 0.12.
I took FH12-12S-1SH(55) FPC flip-lock connector as component reference.

![image](https://cloud.githubusercontent.com/assets/1327611/7050079/62175d0e-de51-11e4-8a5a-ed344e3ef9da.png)

Extended the pads at the side for hand soldering easily and adapt to both flip-lock or insert lock connector.
 
![image](https://cloud.githubusercontent.com/assets/1327611/7050044/23daef74-de51-11e4-8ed3-ae3e6d29a8a7.png)
